content/en/tracing/trace_collection/compatibility/go.md

```diff
@@ -26,12 +26,31 @@ The Go Datadog Trace Library has a [version support policy][2] defined for Go ve
   * [Automatically at compile time using `orchestrion`][78]
   * [Manually add and initialize the Datadog Go tracer][77]
 
+### Go Tracer Support
+
+The Go Tracer v2 is in Preview! See <a href="/tracing/trace_collection/custom_instrumentation/go/migration">the migration guide</a> to upgrade. Support for each version of the Go Tracer is as follows:
+
+| Version	| Preview	   | General Availability (GA)	| Maintenance	| End-of-life (EOL) |
+|---------|------------|----------------------------|-------------|-------------------|
+| v2      | 2024-11-27 | 2025-02-05 (projected)     | TBD         | TBD               |
+| v1      | X          | 2019-02-16                 | 2025-02-05  | 2025-12-31        |
```

Comment:
Did some digging through the release notes/PRs. Is this accurate?

```suggestion
| v2      | 2024-11-27 | 2025-02-05 (projected)     | TBD         | TBD               |
| v1      | 2018-06-06 | 2018-06-06                 | 2025-02-05  | 2025-12-31        |
| v0      | 2016-09-15 | 2016-12-12                 | 2018-06-06  | 2019-06-06        |
```

Reply:
Good finds! I'm not sure if Sept 15, 2016 is a correct date for the preview of v0, though. I see that the first "implementation" of the tracer was made then, but not sure if that would count as a preview "release". For now, I've listed the preview date for v0 to be the same as the GA release date, unless you have found something that I've missed!

Reply:
Sept 15, 2016 was definitely a bit of a stretch 😅  it's fine as the same date